lib/Common/Core/DbgHelpSymbolManager.cpp

```diff
@@ -53,7 +54,7 @@ DbgHelpSymbolManager::Initialize()
 
     // Let's make sure the directory where chakra.dll is, is on the symbol path.
 
-    char16 const * wszModule = AutoSystemInfo::GetJscriptDllFileName();
+    wszModule = AutoSystemInfo::GetJscriptDllFileName();
```

Comment:
So the reason for hoisting this one to the top of the function is because it would allow the variable to be uninitialized in case of `return` or `goto end` in the lines above? Curious, what problem does this cause in real-world code? If it's unused, why require initialization on penalty of compile error?